test/e2e/tests/test_api_keys.py

```
@@ -1046,17 +1046,17 @@ def test_trigger_cleanup_preserves_active_keys(
 
         cleanup_result = sp.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)